text/metrics/0149-exponential-histogram/float64.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package histogram
+
+import (
+	"math"
+)
+
+const (
+	// MantissaWidth is the size of an IEEE 754 double-precision
+	// floating-point mantissa.
+	MantissaWidth = 52
+	// ExponentWidth is the size of an IEEE 754 double-precision
+	// floating-point exponent.
+	ExponentWidth = 11
+
+	// MantissaMask is the mask for the mantissa of an IEEE 754
+	// double-precision floating-point value.
```

Comment:
Peanut gallery comment:  You know it's good when you start bit-shifting and referencing IEEE spec.